tests/test_virtual_chassis.py

```diff
@@ -24,3 +27,184 @@ def test_connectivity(self, vct):
+            metatbl = config_db.get_entry("DEVICE_METADATA", "localhost")
+
+            cfg_switch_type = metatbl.get("switch_type")
+
+            #Test only for line cards
+            if cfg_switch_type == "voq":    
+                lc_switch_id = metatbl.get("switch_id")
+                assert lc_switch_id != "", "Got error in getting switch_id from CONFIG_DB DEVICE_METADATA"
+                if lc_switch_id == "0":
+                    #Testing in Linecard1, In Linecard1 there will be RIF for Ethernet12 from Linecard3 
+                    #Note: Tesing can be done in any linecard for RIF of any system port interface.
+                    #      Here testing is done on linecard with switch id 0
+                    asic_db = DVSDatabase(swsscommon.ASIC_DB, dvs.redis_sock)
+                    keys = asic_db.get_keys("ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFACE")
+                    assert len(keys), "No router interfaces in ASIC_DB"
+
+                    rif_port_oid = ""
+                    for key in keys:
+                        rif_entry = asic_db.get_entry("ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFACE", key)
+                        value = rif_entry.get("SAI_ROUTER_INTERFACE_ATTR_TYPE")
+                        assert value != "", "Got error in getting RIF type"
+                        if value == "SAI_ROUTER_INTERFACE_TYPE_PORT":
+                            value = rif_entry.get("SAI_ROUTER_INTERFACE_ATTR_PORT_ID")
+                            assert value != "", "Got error in getting RIF port"
+                            if value.startswith("oid:0x5d"):
+                                #System port RIF, this is used as key for system port config info retrieval
+                                rif_port_oid = value
+                                break
+
+                    assert rif_port_oid != "", "No RIF records for remote interfaces in ASIC_DB"
+                    #Validate if the system port is from valid switch
+                    sp_entry = asic_db.get_entry("ASIC_STATE:SAI_OBJECT_TYPE_SYSTEM_PORT", rif_port_oid)
+                    value = sp_entry.get("SAI_SYSTEM_PORT_ATTR_CONFIG_INFO")
+                    assert value != "", "Got error in getting system port config info for rif system port"
+                    spcfginfo = ast.literal_eval(value)
+                    #Remote system ports's switch id should not match local switch id
+                    assert spcfginfo["attached_switch_id"] != lc_switch_id, "RIF system port with wrong switch_id"
+
+    def test_chassis_system_neigh(self, vct):
+        """ Test neigh record creation and syncing to chassis app db """
+        
+        """
+        This test validates that:
+           (i)   Local neighbor entry is created with encap index
+           (ii)  Local neighbor is synced to chassis ap db with assigned encap index
+           TODO: (iii) Remote neighbor entry is created in ASIC_DB with received encap index
+        """
```

Comment:
Same feedback as above, but in this case you can just combine this two comments like so:
```suggestion
        """Test neigh record creation and syncing to chassis app db.
        
       This test validates that:
           (i)   Local neighbor entry is created with encap index
           (ii)  Local neighbor is synced to chassis ap db with assigned encap index
           TODO: (iii) Remote neighbor entry is created in ASIC_DB with received encap index
        """
```

Reply:
Fixed. Thanks.